### PR TITLE
[ACCESS-1829] Have docs reflect `standard` deprecation

### PR DESCRIPTION
**Pull request by retsguj**
### What does this PR do?

Updates role-permission docs now that the `standard` permission is deprecated.

### Motivation

https://datadoghq.atlassian.net/browse/ACCESS-1829

### Additional Notes

I've looked through the repo to find more references to the `standard` permission, but haven't found any so far.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/account_management/rbac/permissions.md
Comment (retsguj) [inline content/en/account_management/rbac/permissions.md:44]: Not sure what this meant (users cannot define permissions?), so I removed it.

File: content/en/account_management/rbac/permissions.md
Comment (drichards-87) [reply content/en/account_management/rbac/permissions.md:24]: Are there still users with access to these (now deprecated) permissions? That would be my only concern. Otherwise, I'm fine with removing this section, assuming we don't have some type of deprecation policy where we have to leave notice of it in the docs or elsewhere for a certain period of time.

**Comment by retsguj** [comment]
> Left you a question and approved the PR. Let me know if this is ready to be merged, or if you're waiting for others to review it.

@drichards-87

This is still waiting on review from others on my team.

File: content/en/account_management/rbac/permissions.md
Comment (retsguj) [reply content/en/account_management/rbac/permissions.md:24]: As of now, there still are, but our end goal (which should be accomplished soon) is to remove `standard` from the UI entirely. `admin` already has been removed from the UI.

They both still exist in our database, but from a user's perspective, the `admin` permission does not exist, and `standard` will be the same way soon.

File: content/en/account_management/rbac/permissions.md
Comment (bi1yeu) [inline content/en/account_management/rbac/permissions.md:30]: I might change the first sentence to "All users with one of these roles can read all data types". That's just clarify that the default Read-Only role allows read access to all resources (I think?), but custom roles can be crafted to disallow reading certain resources.

**Review by bi1yeu — APPROVED** [review]
This change makes sense to me, but @sophiah123 and/or @srosenthal-dd should probably sign off

File: content/en/account_management/rbac/permissions.md
Comment (sophiah123) [reply content/en/account_management/rbac/permissions.md:30]: LGTM! Probably in Stephen's queue.

File: content/en/account_management/rbac/permissions.md
Comment (srosenthal-dd) [reply content/en/account_management/rbac/permissions.md:44]: Agree!

**Review by srosenthal-dd — APPROVED** [review]
Love it! Thanks for cleaning it up. This has bugged me for quite a while.

Just so you're aware, there is another task in the backlog with ideas for cleaning up these public docs. If you have more ideas that you didn't get to here, add 'em there: https://datadoghq.atlassian.net/browse/ACCESS-1477

**Comment by drichards-87** [comment]
@retsguj Just checking in again now that a few others have approved the PR. Let me know if it's ready to be merged.

**Comment by retsguj** [comment]
> @retsguj Just checking in again now that a few others have approved the PR. Let me know if it's ready to be merged.

@drichards-87 It is ready now! Thank you

**Pull request merged**
